pl_bolts/models/self_supervised/simclr/transforms.py

```diff
@@ -79,50 +119,90 @@ class SimCLREvalDataTransform(object):
+        jitter_strength: float = 1.,
+        normalize=None
+    ):
+        super().__init__(
+            normalize=normalize,
+            input_height=input_height,
+            gaussian_blur=gaussian_blur,
+            jitter_strength=jitter_strength
+        )
+
+        # replace online transform with eval time transform
+        self.online_transform = transforms.Compose([
+            transforms.Resize(int(self.input_height + 0.1 * self.input_height)),
+            transforms.CenterCrop(self.input_height),
+            self.final_transform,
+        ])
 
-        self.s = s
+
+class SimCLRFinetuneTransform(object):
+    def __init__(
+        self,
+        input_height: int = 224,
+        jitter_strength: float = 1.,
+        normalize=None,
+        eval_transform: bool = False
+    ) -> None:
+
+        self.jitter_strength = jitter_strength
         self.input_height = input_height
-        self.test_transform = transforms.Compose([
-            transforms.Resize(input_height + 10, interpolation=3),
-            transforms.CenterCrop(input_height),
-            transforms.ToTensor(),
-        ])
+        self.normalize = normalize
+
+        self.color_jitter = transforms.ColorJitter(
+            0.8 * self.jitter_strength,
+            0.8 * self.jitter_strength,
+            0.8 * self.jitter_strength,
+            0.2 * self.jitter_strength
+        )
+
+        if not eval_transform:
+            data_transforms = [
+                transforms.RandomResizedCrop(size=self.input_height),
+                transforms.RandomHorizontalFlip(p=0.5),
+                transforms.RandomApply([self.color_jitter], p=0.8),
+                transforms.RandomGrayscale(p=0.2)
+            ]
+        else:
+            data_transforms = [
+                transforms.Resize(int(self.input_height + 0.1 * self.input_height)),
+                transforms.CenterCrop(self.input_height)
+            ]
+
+        if normalize is None:
+            final_transform = transforms.ToTensor()
+        else:
+            final_transform = transforms.Compose([transforms.ToTensor(), normalize])
+
+        data_transforms.append(final_transform)
+        self.transform = transforms.Compose(data_transforms)
 
     def __call__(self, sample):
-        transform = self.test_transform
-        xi = transform(sample)
-        xj = transform(sample)
-        return xi, xj
+        return self.transform(sample)
 
 
 class GaussianBlur(object):
-    """
-    Implements Gaussian blur as described in the SimCLR paper
-    """
-
-    def __init__(self, kernel_size: int, min: float = 0.1, max: float = 2.0):
-        if not _TORCHVISION_AVAILABLE:
-            raise ModuleNotFoundError(  # pragma: no-cover
-                'You want to use `transforms` from `torchvision` which is not installed yet.'
-            )
-
+    # Implements Gaussian blur as described in the SimCLR paper
+    def __init__(self, kernel_size, p=0.5, min=0.1, max=2.0):
```

Comment:
```suggestion
    def __init__(self, kernel_size: int, p: float = 0.5, min: float = 0.1, max: float = 2.0):
```
pls use longer var then `p` and do not overwrite native func like `min/max`